forge-gui/res/cardsfolder/upcoming/cone_of_cold.txt

```diff
@@ -8,6 +8,6 @@ SVar:DBNoUntap:DB$ PumpAll | ValidCards$ Creature.OppCtrl | KW$ HIDDEN This card
 SVar:TapLock2:DB$ TapAll | ValidCards$ Creature.OppCtrl | SubAbility$ NoUntap2 | StackDescription$ SpellDescription | SpellDescription$ 20 VERT Tap all creatures your opponents control. Those creatures don't untap during their controllers' next untap steps.
 SVar:NoUntap2:DB$ PumpAll | ValidCards$ Creature.OppCtrl | KW$ HIDDEN This card doesn't untap during your next untap step. | IsCurse$ True | Duration$ Permanent | SubAbility$ DBEffect
 SVar:DBEffect:DB$ Effect | TgtZone$ Battlefield | ValidCards$ Creature.OppCtrl | ReplacementEffects$ TapLockETBTapped | Name$ Cone of Cold Effect | Duration$ UntilYourNextTurn | StackDescription$ SpellDescription | SpellDescription$ 20 VERT Tap all creatures your opponents control. Those creatures don't untap during their controllers' next untap steps. Until your next turn, creatures your opponents control enter the battlefield tapped.
-SVar:TapLockETBTapped:Event$ Moved | ValidCard$ Creature.OppCtrl | Destination$ Battlefield | ReplaceWith$ ETBTapped | ActiveZones$ Battlefield | SubAbility$ TapAndLock
-SVar:ETBTapped:DB$ ChangeZone | Origin$ All | Destination$ Battlefield | Tapped$ True | Defined$ ReplacedCard
+SVar:TapLockETBTapped:Event$ Moved | ValidCard$ Creature.OppCtrl | Destination$ Battlefield | ReplaceWith$ ETBTapped | ReplacementResult$ Updated | SubAbility$ TapAndLock
```

Comment:
also remove the `| SubAbility$ TapAndLock` part? ReplacementEffects doesn't have SubAbility